tensorrt_llm/_torch/modules/mamba/mamba2_mixer.py

```
@@ -282,6 +283,67 @@ def _try_attach_nvfp4_scale(self):
+                chunk_offsets=chunk_offsets,
+                dt_softplus=self.delta_softplus,
+                dt_limit=(0.0, float("inf")),
+                cu_seqlens=cu_seqlens,
+                seq_idx=seq_idx,
+                return_varlen_states=True,
+                return_final_states=False,
+                out=out_buf,
+                state_dtype=state_dtype,
+            )
+            torch.cuda.synchronize()
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)